modules/sdk/project-templates/project-templates-rest-builder/src/main/resources/archetype-resources/__artifactId__-impl/build.gradle

```diff
@@ -27,6 +27,7 @@ dependencies {
 
 	restBuilder (group: "com.liferay", name: "com.liferay.portal.tools.rest.builder", version: "1.0.93") {force = true}
 #else
+ 	compileOnly group: "javax.annotation", name: "javax.annotation-api", version: "1.3.2"
```

Comment:
please move this below com.liferay.portal. dependencies are sorted alphabetically when possible